src/applications/veteran-id-card/containers/EmailCapture.jsx

```diff
@@ -13,7 +13,7 @@ class EmailCapture extends React.Component {
 
   handleSubmit(e) {
     e.preventDefault();
-    this.props.submitEmail(this.props.email.value);
+    this.props.submitEmail(this.props.email);
```

Comment:
Email was previously an object, now is a string. Do not need to access `.value` anymore